From 394edc1d874099f42f4a6cbd11966727cb92c71f Mon Sep 17 00:00:00 2001
From: Eli Nidam <elini@marvell.com>
Date: Sun, 21 Jul 2013 15:57:54 +0300
Subject: [PATCH 0856/1825] axp: update PHY address for AMC board related to
 latest ECO

https://github.com/MISL-EBU-System-SW/misl-windriver.git linux-3.4.69-14t2-read
commit f2402469c6cc4355c18a1b4d4ca2964067f43026

	Change PHY address for AXP-AMC board port 1,2 from 0x0E, 0x0F to 0x0C, 0x0D

Change-Id: I1874464550e9018688b257d648e7accf3eddedd9
Signed-off-by: Eli Nidam <elini@marvell.com>
Reviewed-on: http://vgitil04.il.marvell.com:8080/2740
Reviewed-by: Nadav Haklai <nadavh@marvell.com>
Tested-by: Nadav Haklai <nadavh@marvell.com>
Signed-off-by: Zhong Hongbo <hongbo.zhong@windriver.com>
---
 .../armada_xp_family/boardEnv/mvBoardEnvSpec.c     |    8 ++++----
 1 files changed, 4 insertions(+), 4 deletions(-)

diff --git a/arch/arm/mach-armadaxp/armada_xp_family/boardEnv/mvBoardEnvSpec.c b/arch/arm/mach-armadaxp/armada_xp_family/boardEnv/mvBoardEnvSpec.c
index 126240d..9aafd57 100755
--- a/arch/arm/mach-armadaxp/armada_xp_family/boardEnv/mvBoardEnvSpec.c
+++ b/arch/arm/mach-armadaxp/armada_xp_family/boardEnv/mvBoardEnvSpec.c
@@ -1085,10 +1085,10 @@ MV_BOARD_TWSI_INFO	db78X60amcInfoBoardTwsiDev[] = {
 
 MV_BOARD_MAC_INFO db78X60amcInfoBoardMacInfo[] = {
 	/* {{MV_BOARD_MAC_SPEED	boardMacSpeed, MV_U8 boardEthSmiAddr}} */
-	{BOARD_MAC_SPEED_AUTO, 0x1,0x0,0x0},
-	{BOARD_MAC_SPEED_AUTO, 0xF,0x0,0x0},
-	{BOARD_MAC_SPEED_AUTO, 0xE,0x0,0x0},
-	{BOARD_MAC_SPEED_AUTO, 0x0,0x0,0x0}
+	{BOARD_MAC_SPEED_AUTO, 0x1, 0x0, 0x0},
+	{BOARD_MAC_SPEED_AUTO, 0xD, 0x0, 0x0},
+	{BOARD_MAC_SPEED_AUTO, 0xC, 0x0, 0x0},
+	{BOARD_MAC_SPEED_AUTO, 0x0, 0x0, 0x0}
 };
 
 
-- 
1.7.5.4

